packages/fxa-settings/src/pages/ResetPasswordConfirmed/index.test.tsx

```diff
@@ -51,8 +49,8 @@ describe('ResetPasswordConfirmed', () => {
 
     fireEvent.click(passwordResetContinueButton);
     expect(logViewEvent).toHaveBeenCalledWith(
-      'settings.reset-password-confirmed.continue',
-      'reset_password_confirmed_continue',
+      'reset-password-confirmed',
+      'reset-password-confirmed.continue',
```

Comment:
Can we also test the `entrypoint_variation: 'react',` here?

Reply:
I think it does test that, in the lines below? Should I change the way that I test for it?

Reply:
Sorry, you right, I don't know how I missed that 🤦‍♀️